README.md

```diff
@@ -70,3 +72,29 @@ To debug the app from within the template, run the following commands:
    ```ps
    flutter run
    ```
+
+### Command Line Interface
+
+To debug the CLI, do the following:
+
+1. Go to the CLI directory.
+   ```ps
+   cd src/cli
+   ```
+1. Restore the packages.
+   ```ps
+   flutter pub get
+   ```
+1. Run the code generators.
+   ```ps
+   dart run build_runner build
+   ```
+1. Set the variables value in `src/cli/lib/src/commands/create_command.dart` that would normally be done by the Pipeline
+   - `_commitHash`  
+   - `_shortCommitHash`
+   - `_versionNumber`
+   - `_commitDate`
+   > 💡 You have to set real values, you should probably use the latest commit information or your own commit from your branch.
```

Comment:
Can you give a full example with an existing commit so that we can see the format?

Reply:
No specific format, it's just a string displayed.